### ISSUE TITLE
Google getUserAttributes

### ISSUE THREAD

**Issue by Julian-B90** [2amigos/yii2-usuario#316]
### What steps will reproduce the problem?

Google OAuth an click login with google

### What is the expected result?
The Code expected an array from emails (https://github.com/2amigos/yii2-usuario/blob/master/src/User/AuthClient/Google.php#L24)

### What do you get instead?
But i get an single email property and the `$this->client->getEmail()` was empty

This is what i get from `$this->client->getUserAttributes()`
```
Array ( 
[id] => 108xxx
[email] => julianxxx@gmail.com
[verified_email] => 1 
[name] => Juxx Bxxx 
[given_name] => Juxx 
[family_name] => Bxxx
[picture] => xxxxxy1yM/photo.jpg 
[locale] => de )
```

**Comment by tonydspaniard** [comment]
Seems the response had changed. Will double check from OAuth extension. Thanks for reporting.

**Comment by Julian-B90** [comment]
@tonydspaniard i have made a PR. This supports the new response and the old response.

**Comment by tonydspaniard** [comment]
Awesome!

**Comment by tonydspaniard** [comment]
This has been fixed by #318
